addons/core/translations/resources/en-us.yaml

```diff
@@ -459,6 +459,9 @@ credential-library:
   types:
     vault-generic: Generic Secrets
     vault-ssh-cert: SSH Certificate Credentials
+  help:
+    vault-generic: Authorize sessions using credentials brokered by Vault
```

Comment:
Can we describe about the type without talking about authorization or sessions?

Reply:
Do you have any suggestions? I'm not sure what else it would say

Reply:
I'm going to defer any rewording for later once I can get some more input from others as well and merge it for now